decentraland/unity-explorer
# feat: map discoverability

This PR contains all the changes for the map discoverability shape up

Testable things so far:
1) Categories filter (UI might not be final and will need to be moved)
* On top of the map we have the categories filter
* Verify that only one category at a time can be active
* Verify that we can disable the only active category
* Verify all icons appear as expected when enabling them

2) Map type switching
* Launch the explorer
* Open the map
* Open the filter view in bottom left
* Switch between parcel view and satellite view

3) Toggle specific layer panels
* Launch the explorer
* Open the map
* Verify that the new button in bottom left opens and closes the layer panel
* Verify that the layers minigames, POIs ,favorites, people are working when toggling them on and off

4) Map Pins clustering
* Launch the explorer
* Open the map
* Verify that at different zoom levels closeby icons get clustered together

5) Places panel
* Try search for a place (for example genesis)
* All the places displayed in the panel should also be visible as pins in the map
* Click any place and check that the information is displayed correctly
* Check for photos tab (genesis plaza has some photos)
* Check events tab (genesis plaza has some events)
* Click on any event
* Check that the event information is displayed correctly
* Use the back button in the search bar. It should go to the previous screen
